### DISCUSSION TITLE
install Advance Netconf Explorer

### DISCUSSION THREAD

**Discussion by Anajake** [community/community#38223] [Other Feature Feedback, Questions, & Ideas]
Hi Team,

Can you help me to install Advance netconf explorer in Debian GNU/Linux rodete, I have read that it work with docker but only in version debian 10 and 11, and for this reason  I think that I need to run Netconf Using JDK and Maven because the version that I have is Debian GNU/Linux rodete, can you help me, please? I have try many times to install but I can´t.

thanks 
regards

**Comment by ossanna16** [comment]
Hi @Anajake,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Linux. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
